### DISCUSSION TITLE
Does renovate have to run multiple times for rebasing

### DISCUSSION THREAD

**Discussion by alreadyhaveanaccount4** [renovatebot/renovate#20554] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, select which platform you are using.

None

### If you're self-hosting Renovate, tell us what version of the platform you run.

GitLab

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Wanted end result.

 I dont want to run renovate as many times as I have merge conflicts.

### What you tried so far.

So lets assume there are 3 MRs opened by renovate. They all contain changes on the same `package.json` but different groups (e.g. group linters, react and other updates.

When I now merge the first of the 3 MRs, the other ones would have conflicts, because the first one now changed the `package.json` file. I can now check the box "If you want to rebase/retry this MR, check the box", which results in a rebase by renovate and I can merge it again. 

Now here's my question: if this happens for every commit I merge, do I have to run renovate as many times as I have merged MR? Is this expected or am I overseeing something obvious?

**Comment by rarkins** [comment]
If merging one MR causes others to be conflicted, you either need to resolve the conflicts yourself manually, or have Renovate run again and it will do it automatically.I suggest you let the bot do the work. You do not need to tick the checkbox - Renovate will rebase any MR which is conflicted.

**Comment by alreadyhaveanaccount4** [comment, marked as answer]
Thanks for the quick response! 
Yeah, i think i've read  `Renovate will rebase any MR which is conflicted.` in the docs already and this is where my question comes from. Because the Idea was to schedule it via GitLab CI, but wouldn't that mean, that the scheduled run **only creates the MRs** (or automerges existing ones if desired) and whenever I decide to merge one, I need to manually start renovate again?

The Idea was to run renovate once a Week but then some MRs would only be merged after 2 weeks+ if I would let renovate handle it.

The solution would be to run renovate more regularily i guess?

**Reply by rarkins** [reply]
Yes, more regularly, if you want it to be able to rebase

**Reply by alreadyhaveanaccount4** [reply]
Alright, thanks.
